src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
 // write code here
+document.addEventListener('DOMContentLoaded', function () {
+  const listItems = document.querySelectorAll('.tree li');
+
+  listItems.forEach((item) => {
+    const child = item.querySelector('ul');
+
+    if (!child) {
+      return;
+    }
+
+    const span = document.createElement('span');
+
+    span.textContent = item.firstChild.textContent.trim();
+    item.firstChild.replaceWith(span);
+
+    child.style.display = 'block';
+
+    span.addEventListener('click', function (e) {
+      e.stopPropagation();
+
+      if (child.style.display === 'none') {
+        child.style.display = 'block';
+      } else {
+        child.style.display = 'none';
+      }
```

Comment:
Instead of directly changing the style properties, consider using class manipulation. You can define two classes in CSS, one for the visible state and one for the hidden state. Then, you can add and remove these classes based on the click event. This approach has the benefit of keeping the styling separate from the JavaScript code, making it easier to manage and modify in the future.